doc/project/project_roles.rst

```diff
@@ -314,15 +314,48 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals from the list below, including an approval by the
```

Comment:
This sentence now sounds a bit funny. Maybe just:
"Minimal of 2 approvals, including an approval by the designated assignee." (?)